Lib/test/test_sys.py

```diff
@@ -1475,6 +1479,16 @@ def test_gc_head_size(self):
         # but lists are
         self.assertEqual(sys.getsizeof([]), vsize('Pn') + gc_header_size)
 
+    def test_inline_values(self):
```

Comment:
I have overall low confidence that this is the right way to test this, and I'm open to suggestions.